src/guide/frontend/aspnet.md

```diff
@@ -0,0 +1,294 @@
+---
+title: ASP.NET
+---
+# ASP.NET Core 7 Identity
+
+## Getting Started
+The utilization of our NuGet package will streamline the integration process with ASP.NET Identity, facilitating a smoother and more efficient integration experience.
+
+```bash
+dotnet add package Passwordless.AspNetCore
+```
+
+Bootstrapping is now a straightforward process, where the addition of Passwordless to the `IdentityBuilder` becomes a requisite step.
+
+The example below assumes your `appsettings.json` will add all configuration settings under the key `Passwordless`.
+
+```csharp
+builder.Services.AddIdentity<IdentityUser, IdentityRole>()
+    .AddEntityFrameworkStores<PasswordlessContext>()
+    .AddPasswordless(builder.Configuration.GetSection("Passwordless"));
+```
+
+At a minimum, you will be required to furnish the `ApiKey` (public key) and `ApiSecret` (private key).
+
+```json
+{
+  "Passwordless": {
+    "ApiKey": "***:public:***",
+    "ApiSecret": "***:secret:***",
+    "Register": {
+      "Discoverable": true
+    }
+  }
+}
+```
+
+Now, we'll need to modify the `WebApplication` object to add the Passwordless routing to make registering, logging in and managing credentials easier.
+
+The endpoints are documented in a later section on this page.
+
+```csharp
+app.MapPasswordless(true);
```

Comment:
What does the `true` correspond to here? Maybe add a comment

Reply:
Or we can name the argument? `(enableRegisterEndpoint: true)`